packages/aws-appsync/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "aws-appsync",
-  "version": "4.1.1",
+  "version": "4.1.2",
```

Comment:
I think this is not necessary, lerna can do that automatically

Reply:
Will remove!